PhysicsTools/ONNXRuntime/interface/ONNXRuntime.h

```diff
@@ -29,6 +29,8 @@ namespace cms::Ort {
     ONNXRuntime& operator=(const ONNXRuntime&) = delete;
     ~ONNXRuntime();
 
+    static ::Ort::SessionOptions defaultSessionOptions(bool use_cuda = false);
```

Comment:
Maybe an enumeration (`enum class`) would be more self-documenting and extensible?

Reply:
@makortel So something like:

```c++
enum class OrtSessOpt {
  cpu,
  cuda,
};
```

Reply:
Yeah, something like that.